orbit/pkg/useraction/mdm_migration_darwin.go

```diff
@@ -0,0 +1,294 @@
+//go:build darwin
+
+package useraction
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"strconv"
+	"sync"
+	"text/template"
+	"time"
+
+	"github.com/rs/zerolog/log"
+)
+
+type swiftDialogExitCode int
+
+const (
+	primaryBtnExitCode   = 0
+	errorExitCode        = 1
+	secondaryBtnExitCode = 2
+	infoBtnExitCode      = 3
+	timeoutExitCode      = 4
+	userQuitExitCode     = 10
+	unknownExitCode      = 99
+)
+
+var mdmMigrationTemplate = template.Must(template.New("mdmMigrationTemplate").Parse(`
+## Migrate to Fleet
+
+To begin, click "Start." Your default browser will open your My Device page.
+
+{{ if .Aggresive }}You {{ else }} Once you start, you {{ end -}} will see this dialog every 15 minutes until you click "Turn on MDM" and complete the instructions.
```

Comment:
Just a heads-up not to miss this `Aggresive`, as static code analysis will not find it (i.e. it would happily build).